### PR TITLE
Fix small typo

### LINKED ISSUE #2806: Labels have an id="*-label" when referenced by other tags
>  Is it normal no tag has an `id="*-label"`?

_Original comment in https://github.com/LuccaSA/lucca-front/pull/2803#discussion_r1608338246_

### LINKED ISSUE #2807: Fix « invalid » typos
While reviewing a [PR about checkboxes](https://github.com/LuccaSA/lucca-front/pull/2803) some typos were spotted:

https://github.com/search?type=code&q=org%3ALuccaSA+Invlid

### PR DESCRIPTION
**Pull request by vvalentin-lucca**
## Description

- [x] Fix #2806 
- [x] Fix #2807 
- [x] Fix JIT compilation
- [x] Add `type="button"`

-----

Optionally, technical or more in-depth description for reviewers.
Keep an empty line under your text, as well as the 5 lines that follow it.

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2808/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/17300/ui-diff)

File: stories/qa/forms/textfield.stories.html [hunk 1]
```diff
@@ -2,23 +2,23 @@
 	<h2>Sizes</h2>
 
 	<div class="form-field pr-u-marginBottom100">
-		<label class="formLabel" id="fieldSizlabel" for="fieldSiz">
+		<label class="formLabel" id="fieldSizelabel" for="fieldSize">
```
Comment (fbasmaison-lucca) [inline stories/qa/forms/textfield.stories.html:5]: Commenting to create another issue about matching the same pattern `id-topic` or `idTopic`:

- [fieldSizelabel](https://github.com/LuccaSA/lucca-front/pull/2808/files#diff-26a24a9c204e8f59a997764a08ecd24ff4e5e8973b72dba1ca47384c54f3395aR5)
- [fieldXSlabel](https://github.com/LuccaSA/lucca-front/pull/2808/files#diff-26a24a9c204e8f59a997764a08ecd24ff4e5e8973b72dba1ca47384c54f3395aR45).
- [fieldErrorEmptylabel](https://github.com/LuccaSA/lucca-front/pull/2808/files#diff-26a24a9c204e8f59a997764a08ecd24ff4e5e8973b72dba1ca47384c54f3395aR299)
- [fieldErrorFilledlabel](https://github.com/LuccaSA/lucca-front/pull/2808/files#diff-26a24a9c204e8f59a997764a08ecd24ff4e5e8973b72dba1ca47384c54f3395aR319)
- [fieldDisabledEmptylabel](https://github.com/LuccaSA/lucca-front/pull/2808/files#diff-26a24a9c204e8f59a997764a08ecd24ff4e5e8973b72dba1ca47384c54f3395aR351)
- [fieldDisabledFilledlabel](https://github.com/LuccaSA/lucca-front/pull/2808/files#diff-26a24a9c204e8f59a997764a08ecd24ff4e5e8973b72dba1ca47384c54f3395aR372)
- [fieldAlllabel](https://github.com/LuccaSA/lucca-front/pull/2808/files#diff-26a24a9c204e8f59a997764a08ecd24ff4e5e8973b72dba1ca47384c54f3395aR402)

The `(id|by)=".+[^-](label|message)"` regexp can help.

**Review by fbasmaison-lucca — DISMISSED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2808/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/17681/ui-diff)

**Review by jeremie-lucca — APPROVED** [review]

**Pull request merged**